urdf/CMakeLists.txt

```diff
@@ -22,7 +22,20 @@ set(generated_compat_header "${CATKIN_DEVEL_PREFIX}/include/${PROJECT_NAME}/urdf
 include_directories("${CATKIN_DEVEL_PREFIX}/include")
 configure_file(urdfdom_compatibility.h.in "${generated_compat_header}" @ONLY)
 
-add_compile_options(-std=c++11)
```

Comment:
Instead of going to all of the trouble with these flags, I'm wondering if we should just remove this compile option completely.  We have a separate `melodic-devel` branch here, and all of the compilers that are supported (g++ 7.4.0 on Bionic, g++ 6.3.0 on Stretch, and now MSVC) use C++14 by default.  Thus, I'm not sure that overriding their defaults is a good idea.  @sloretz thoughts?